component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/internal/TOTPAuthenticatorServiceComponent.java

```diff
@@ -106,4 +110,12 @@ protected void setRealmService(RealmService realmService) {
 	protected void unsetRealmService(RealmService realmService) {
 		TOTPDataHolder.getInstance().setRealmService(null);
 	}
-}
\ No newline at end of file
+
+	protected void unsetIdentityEventService(IdentityEventService eventService) {
+		TOTPDataHolder.getInstance().setIdentityEventService(null);
```

Comment:
If we always set "null" do we need to pass in "IdentityEventService eventService" ?